src/views/notification/index.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+use yii\helpers\Html;
+use kartik\grid\GridView;
+Use kartik\editable\Editable;
+use webzop\notifications\model\NotificationType;
+use kartik\daterange\DateRangePicker;
+
+/* @var $this yii\web\View */
+/* @var $searchModel webzop\notifications\model\NotificationSearch */
+/* @var $dataProvider yii\data\ActiveDataProvider */
+
+
+$this->title = 'Notifications';
+$this->params['breadcrumbs'][] = $this->title;
+?>
+<div class="notifications-index">
+    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js" crossorigin="anonymous"></script>
```

Comment:
this should be a requirement in your composer, not specified directly in the view. Use: https://github.com/kartik-v/yii2-icons